app/templates/community/meetups-getting-started.hbs

```diff
@@ -81,7 +81,13 @@
   <h3>Some other tidbits</h3>
   <ul>
     <li>
-      For more content like this, <a href="https://medium.com/@wifelette/the-bits-n-bobs-of-starting-a-user-group-a44cfdd6bed9">check out @wifelette's blog post</a>; much of the same content, but some new tips and tricks.
+      For more content like this,
+      <a
+        href="https://medium.com/@wifelette/the-bits-n-bobs-of-starting-a-user-group-a44cfdd6bed9"
+      >
+        check out @wifelette's blog post
+      </a>
+      ; much of the same content, but some new tips and tricks.
```

Comment:
This is adding a space: 
![screen shot 2018-10-26 at 09 14 34](https://user-images.githubusercontent.com/594890/47553688-97cf3780-d8ff-11e8-9197-a402a80f4d58.png)